src/test/ui/closure-expected-type/expect-region-supply-region.nll.stderr

```diff
@@ -49,7 +49,7 @@ LL |     let mut f: Option<&u32> = None;
    |         ----- lifetime `'2` appears in the type of `f`
 ...
 LL |     closure_expecting_bound(|x: &'x u32| {
-   |                              - lifetime `'1` appears in this argument
+   |                                 - let's call the lifetime of this reference `'1`
```

Comment:
I hadn't noticed this before, shouldn't it complain about `'x` needing to outlive `'2`, pointing out that it is unconstrained on the input so it evaluates shorten than `'2`? I found the message surprising until I looked at the code.